apps/tup-cms/src/apps/portal/apps.py

```diff
@@ -79,11 +80,21 @@ def send_confirmation_email(form_name, form_data):
     [form_data["email"]],
     html_message=email_body)
 
-
-def callback(form, cleaned_data, **kwargs):
-    logger.debug(f"received submission from {form.name}")
+def add_user_hetdex_allocation(request):
+    headers = {"x-tup-token": settings.TUP_SERVICES_ADMIN_JWT}
+    data = {"username": request.user.username}
+    admin_params = {"username": "admin"}
+    requests.post(f"{service_url}/projects/{hetdex_allocation}/users",
```

Comment:
One of the requirements here is to "email Maytal with the user info and form data" so I think that needs to happen in this method as well. We should probably also be sending the standard confirmation email to users (rt-ticket-form doesn't do this because RT emails the user instead of us).

Reply:
Marking resolved, didn't realize this was configurable in CMS.